components/Forms/Field/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Field';
```

Comment:
Is this needed for Storybook or is there another reason for this? Sorry if I'm missing something obvious here 😬

Reply:
It's a way of simplifying imports when you use fishtown-ui! It really just bubbles up the imports to the top level.

For example:

Without these index export's:
`import { Icon } from 'fishtown-ui/components/Icon/Icon';`

With:
`import { Icon } from 'fishtown-ui';`